spec/safe_statements_spec.rb

```diff
@@ -877,7 +931,7 @@ def up
           end
 
           it "raises if connecting to Postgres 9.1 databases" do
-            allow(ActiveRecord::Base.connection).to receive(:postgresql_version).and_return(90112)
+            allow(ActiveRecord::Base.connection).to receive(:postgresql_version).and_return(9_01_12)
```

Comment:
This is reasonable cleanup for understandability, but I think it should be a standalone commit.

Reply:
Moving to its own pr.